apps/src/code-studio/pd/workshop_enrollment/workshop_enroll.jsx

```diff
@@ -155,55 +152,12 @@ export default class WorkshopEnroll extends React.Component {
   }
 
   renderSuccess() {
-    if (this.props.workshop.course === COURSE_BUILD_YOUR_OWN) {
-      // If successfully enrolled in Build Your Own workshop, redirect to My PL landing page. The
-      // WORKSHOP_ENROLLMENT_COMPLETED_EVENT event will be logged on that page since event logs
-      // immediately followed by redirects sometimes do not fire.
-      const rpName = this.props.workshop.regional_partner?.name;
-      const wsCourse = this.props.workshop.course;
-      const wsSubject = this.props.workshop.subject;
-      window.location.href = `/my-professional-learning?rpName=${rpName}&wsCourse=${wsCourse}&wsSubject=${wsSubject}`;
-    } else {
-      analyticsReporter.sendEvent(EVENTS.WORKSHOP_ENROLLMENT_COMPLETED_EVENT, {
```

Comment:
Should this event be deleted?

Reply:
Oh I should've clarified sorry, this utilizes previous work in [this PR](https://github.com/code-dot-org/code-dot-org/pull/59768) to use enrollment params to [log that event](https://github.com/code-dot-org/code-dot-org/pull/59768/files#diff-bbd34377c486e39c92e30fec93c154c47a637f5e06991ef7f505de838529b42dR94-R98).

Reply:
oh perfect!